packages/testing/containers/services/registry.ts

```diff
@@ -0,0 +1,37 @@
+import { gitea, createGiteaHelper } from './gitea';
+import { keycloak, createKeycloakHelper } from './keycloak';
+import { loadBalancer } from './load-balancer';
+import { mailpit, createMailpitHelper } from './mailpit';
+import { createObservabilityHelper } from './observability';
+import { postgres } from './postgres';
+import { proxy } from './proxy';
+import { redis } from './redis';
+import { taskRunner } from './task-runner';
+import { tracing, createTracingHelper } from './tracing';
+import type { Service, ServiceResult, HelperFactories } from './types';
+import { vector } from './vector';
+import { victoriaLogs } from './victoria-logs';
+import { victoriaMetrics } from './victoria-metrics';
+
+export const services: Record<string, Service<ServiceResult>> = {
```

Comment:
Should this be defined as a const so it has the exact shape and not Record with any key?